css-fonts-4/Overview.bs

```diff
@@ -2908,6 +2908,71 @@ Initial: normal
 
 This descriptor defines initial settings that apply when the font defined by an @font-face rule is rendered. It does not affect font selection. Values are identical to those defined for the 'font-language-override!!property' property defined below except that the value inherit is omitted. When multiple font feature descriptors, properties, or variations are used, the cumulative effect on text rendering is detailed in the section [[#font-feature-variation-resolution]] below.
 
+<h3 id="font-metrics-override-desc">
+Default font metrics overriding:
+the 'ascent-override', 'descent-override' and 'line-gap-override' descriptors</h3>
+
+<pre class='descdef'>
+Name: ascent-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+<pre class='descdef'>
+Name: descent-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+<pre class='descdef'>
+Name: line-gap-override
+Value: normal | <<percentage>>
+For: @font-face
+Initial: normal
+</pre>
+
+The 'ascent-override', 'descent-override' and 'line-gap-override' descriptors define the
+<a spec="CSS-INLINE-3">ascent metric</a>, <a spec="CSS-INLINE-3">descent metric</a> and
+<a spec="CSS-INLINE-3">line gap metric</a> of the font.
+
+When the descriptor value is 'normal', the corresponding metric value is obtained from the
+font file directly.
+
+Note: User agents may choose different pieces of data from the font file as the metric value,
+leading to different layouts. For example, Chrome on Windows uses sTypoAscender, sTypoDescender
```

Comment:
I don't think there's precedence for mentioning one specific browser inside a CSS spec.

Reply:
> For example, Chrome on Windows uses sTypoAscender, sTypoDescender
> [... and every browser does something different on every platform]

I really wish the spec would require all browsers on all platforms to do the same thing (I know that's a red flag for some people, but it would be opt-in), in order to solve the issue #2228 properly (without requiring web developers to figure out and supply percentages).

Reply:
Yes, I think it is enough to mention that different implementations on different platforms use metrics drawn from different places in the font.

Reply:
Revised accordingly.